plugin/tests/e2e/config/bootstrap.js

```diff
@@ -205,8 +205,8 @@ beforeAll( async () => {
 		width: 1680,
 		height: 948,
 	} );
-	await page.setDefaultNavigationTimeout( 10000 );
-	await page.setDefaultTimeout( 10000 );
+	await page.setDefaultNavigationTimeout( 100000 );
```

Comment:
whats the reason for the increase 10x?

Reply:
@rodydavis Error on my part, my computer keeps timing out on tests, I'll revert this